src/styles/navbar.module.scss

```
@@ -9,9 +12,13 @@
   height: 100px;
 }
 
+.loginText{
+  color: var(--text-main);
+}
 .topnav_mobile {
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  grid-template-columns: 30px 1fr auto;
```

```suggestion
  grid-template-columns: auto 1fr auto;
```
I think this would be better. 🤔